source/demo-app/RssFeed.cpp

```diff
@@ -1,52 +1,26 @@
 #include "RssFeed.h"
 #include <iostream>
 
-RssFeed* RssFeed::instance = nullptr;
-
 RssFeed::RssFeed(QObject* parent)
     : QObject(parent)
 {
-   
-}
-
-RssFeed& RssFeed::getInstance(){
-    if (instance == nullptr){
-        instance = new RssFeed();
-    }
-    return *instance;
-}
+    connect(&m_refreshTimer, &QTimer::timeout, this, &RssFeed::refreshFeed);
 
-QStringList RssFeed::getEntries() const {
-    return entries;
+    QTimer::singleShot(200, &RssFeed::refreshFeed);
 }
 
-void RssFeed::clearFeed(const v8::FunctionCallbackInfo<v8::Value>& args) {
-     getInstance().entries.clear();
+void RssFeed::refreshFeed() {
+    m_entries.clear();
+    node::lib::call("emitRequest",
+                    'http://feeds.bbci.co.uk/news/world/rss.xml',
+                    [this](v8::Local<v8::Value> entry){ addEntry(entry); },
+                    [this](){ emit entriesChanged(); } );
```

Comment:
which call() definition is this? the signatures look different in what i believe to be the [newest version](https://github.com/hpicgs/node/blob/node_lib/src/nodelib.h) of our API draft, which expects a vector of args, and not just as many arguments as one wants.

We also might need to revisit the "call C++ function from JS" use case in our API for these lambda arguments to work.